projects/kit/src/lib/masks/number/processors/thousand-separator-postprocessor.ts

```diff
@@ -22,21 +22,21 @@ export function createThousandSeparatorPostprocessor({
         return identity;
     }
 
-    const prefixReg = new RegExp(`^${escapeRegExp(prefix)}${CHAR_MINUS}?`);
-    const postfixReg = new RegExp(`${escapeRegExp(postfix)}$`);
     const isAllSpaces = (...chars: string[]): boolean => chars.every(x => /\s/.test(x));
 
     return ({value, selection}) => {
-        const [integerPart, decimalPart = ''] = value.split(decimalSeparator);
+        const {cleanValue, extractedPostfix, extractedPrefix} = extractPrefixAndPostfix({
+            value,
+            prefix,
+            postfix,
+            shouldExtractMinus: true,
```

Comment:
Is it the only place where we use `shouldExtractMinus` ? 🤔 

I think we should **not** incapsulate this logic inside `extractPrefixAndPostfix` and we can extract the possible minus sign independently by separate RegExp inside this file.